### ISSUE TITLE
Interface Improvements

### ISSUE THREAD

**Issue by nomisum** [gruppe-adler/ar3play-web#4]
First Draft/Proposal
![interface](https://cloud.githubusercontent.com/assets/500164/6986755/bf81577a-da40-11e4-9d59-866580e8b86a.jpg)

**Comment by Dahlgren** [comment]
Looks nice! Maybe have the map have full width? I guess it should be possible to display the three fighting sides, maybe civilian as well?

**Comment by nomisum** [comment]
yeah, good catch. dimensions are heavily inspired by current implementation :D

**Comment by Dahlgren** [comment]
Should the selected map filter the missions selection? Since the map is known for each mission it should (hopefully :D) not be needed to display the correct map.

**Comment by nomisum** [comment]
Correct, IMHO map/replay selection should be merged into one control. Select mission and then filter by date/map/missionname

Some thinking still tbd all in all :)

**Comment by Dahlgren** [comment]
Did a quick cleanup of the current UI, it now fills the screen correctly to avoid scrolling etc. Moved all settings to a modal dialog.
![screencapture-file-ar3play-web](https://cloud.githubusercontent.com/assets/288631/6990548/763c3854-da6e-11e4-97df-3d286b2df48f.png)

I've also looked at a couple of angular map libraries

**Comment by Fusselwurm** [comment]
^ merged, see #5

**Comment by Dahlgren** [comment]
I'll try to get started on an Angular rewrite with some more modularised behaviour and pages
